app/views/layouts/_navigation.html.erb

```diff
@@ -9,6 +9,9 @@
               <li class="nav-item">
                 <button class="btn btn-outline-secondary mr-1"><a href="/obituaries/new">Add Obituary</a></button>
               </li>
+              <li class="nav-item">
+                <button class="btn btn-outline-secondary mr-1" ><a id="profile" href="/profile">Profile</a></button>
```

Comment:
Added Profile Button to nav bar, which is getting a bit crowded. Might have to discuss cleaning this up in the future.

Reply:
Good call.

Reply:
Nice work!